relayer/metrics/metricsService.go

```diff
@@ -89,11 +89,16 @@ func prepareArrayForProject(projectData map[string]map[string]*AggregatedMetric,
 	var toSendData []RelayAnalyticsDTO
 	for chainKey, chainData := range projectData {
 		for apiTypekey, apiTypeData := range chainData {
+			var averageLatency int64
+			if apiTypeData.SuccessCount > 0 {
+				averageLatency = int64(apiTypeData.TotalLatency / uint64(apiTypeData.SuccessCount))
```

Comment:
mixing int and uint is a bad practice

Reply:
also, it should be a float...

Reply:
sure i can make it all uint, but as for float its not necessary here since we are talking about ms (the floating part would be nano s ). do we need that precise ?